example/src/pie/PieChartBasic.js

```diff
@@ -63,12 +64,33 @@ class PieChartBasic extends Component {
     }
 
     return (
-      <View>
-        <Pie
-          data={data}
+      <ScrollView horizontal={true} style={{flex:1,backgroundColor:'#F5FCFF'}} contentContainerStyle={{justifyContent:'center',alignItems:'center'}}>
```

Comment:
Is a ScrollView actually needed here? (i've not opened example app just yet to view this)